update (fix autoWalk)

poxis opened the pull request:
Fix 'new' autoWalk system (not updated features when adding this system)
Edit:  **bug fix?** and **Merge branch 'mehah:main' into cache-for-all** is old :|

poxis committed:
bug fix?

I don't know about everyone, but over the long way, the customer didn't keep looking for the road if they encountered a creature.

poxis committed:
Merge branch 'mehah:main' into cache-for-all

poxis committed:
update (fix autoWalk)

poxis committed:
update(fix autoWalk)

poxis committed:
update (fix autoWalk)

poxis committed:
not added, but nice option

Znote reviewed (approved):
Tested, seems to work great

poxis committed:
minimap lite bug

did not refresh the grids if there was a change floor item [eg. itemid: 406 to 407] (and has mimimapColor = 0, or none.)

oarsc committed:
Merge branch 'main' into cache-for-all

oarsc commented:
Merged 👍🏼
